evap/student/templates/student_vote.html

```diff
@@ -90,77 +109,79 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
+                        {% endfor %}
+                    </div>
                 </div>
-            </div>
-        {% endif %}
-        {% if small_evaluation_size_warning and not preview and evaluation.num_voters == 0 %}
-            <div class="callout callout-warning mb-3" id="bottom_text_results_publish_confirmation_card">
-                <div class="callout-header">
-                    <span class="fas fa-triangle-exclamation"></span> {% translate 'Small number of participants' %}
+            {% endif %}
+            {% if evaluation_form_group_bottom %}
+                <div class="card card-outline-primary mb-3">
+                    <div class="card-header">
+                        {% translate "General questions" %}
+                    </div>
+                    <div class="card-body">
+                        {% include "student_vote_questionnaire_group.html" with questionnaire_group=evaluation_form_group_bottom textanswers_visible_to=general_contribution_textanswers_visible_to preview=preview %}
+                    </div>
                 </div>
-                <div class="callout-content tab-row">
-                    {% blocktranslate %}You're the first person taking part in this evaluation. If you want your text answers to be shown to the contributors even if you remain the only person taking part, please check the box below. Otherwise your text answers will be deleted if no other person takes part in the evaluation.{% endblocktranslate %}
-
-                    <div class="form-check mt-3">
-                        <input class="form-check-input tab-selectable" id="text_results_publish_confirmation_bottom" type="checkbox"
-                            name="text_results_publish_confirmation_bottom" />
-                        <label class="form-check-label" for="text_results_publish_confirmation_bottom">
-                            {% translate 'Show my text answers to the contributors even if I remain the only person who takes part in this evaluation.' %}
-                        </label>
+            {% endif %}
+            {% if small_evaluation_size_warning and not preview and evaluation.num_voters == 0 %}
+                <div class="callout callout-warning mb-3" id="bottom_text_results_publish_confirmation_card">
+                    <div class="callout-header">
+                        <span class="fas fa-triangle-exclamation"></span> {% translate 'Small number of participants' %}
+                    </div>
+                    <div class="callout-content tab-row">
+                        {% blocktranslate %}You're the first person taking part in this evaluation. If you want your text answers to be shown to the contributors even if you remain the only person taking part, please check the box below. Otherwise your text answers will be deleted if no other person takes part in the evaluation.{% endblocktranslate %}
```

Comment:
This warning is inside the `{{ language ... }}` block, while the one at the top of the site is not. This means that the same warning can be displayed in two different languages.

In general, it is a bit odd that the between the language toggle for the evaluation and the evaluation form there is still a warning and the "Thank you for taking part ..." message which are displayed in the site-wide language. Should we just translate those too? @janno42

Reply:
We specifically decided that only the questionnaire section should be affected, the rest of the page stays in the selected page language. The fix in [7ecc418](https://github.com/e-valuation/EvaP/pull/2367/commits/7ecc418200e20c542a1feda64786d8db92698a1a) is fine.